packages/hooks/src/ContextProvider/controllers/Pipeline.ts

```diff
@@ -69,11 +67,19 @@ export class Pipeline {
       p.version = name.version;
     }
 
+    const clientConfig: { [key: string]: string } = {};
+    if (typeof config.userAgent !== 'undefined') {
```

Comment:
I think you can use `isNullOrUndefined` function from the `utils` package.

```suggestion
    if (!isNullOrUndefined(config.userAgent)) {
```